projects/prison-case-notes-to-probation/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/model/DeliusCaseNote.kt

```diff
@@ -42,16 +42,20 @@ data class CaseNoteBody(
 
     fun description(caseNoteType: CaseNoteType): String? {
         return when {
-            typeLookup().isAlertType() -> "NOMIS ${
-                if (content.length > 194) "${content.take(192)} ~" else content.take(194)
-            }"
-
+            typeLookup().isAlertType() -> "NOMIS $content".truncatedDescription()
             isResettlementPassport() -> BcstPathway.from(content)?.let { "BCST case note for $it" }
             caseNoteType.code == CaseNoteType.DEFAULT_CODE -> "NOMIS Case Note - $type - $subType"
             else -> null
         }
     }
 
+    private fun String.truncatedDescription(): String {
+        val bytes = this.toByteArray()
+        return if (bytes.size > 200) {
+            String(bytes.slice(0..197).toByteArray()) + " ~"
```

Comment:
Is there a chance of us corrupting the end of the text here, if we remove one byte from a multi-byte character?  Not sure how best to deal with that...